plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSinkProvider.java

```diff
@@ -122,7 +143,12 @@ public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHa
                         session,
                         optimizeHandle.getFileFormat(),
                         optimizeHandle.getTableStorageProperties(),
-                        maxOpenPartitions);
+                        maxOpenPartitions,
+                        optimizeHandle.getSortOrder(),
```

Comment:
I see you pass these 3 fields in many places, maybe it would make sense to create an object (a record) to represent them ? Similar to SortingFileWriterConfig ? WDYT ?